src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -103,7 +99,8 @@ public static CassandraDaemon getInstanceForTesting()
         return instance;
     }
 
-    static {
+    static
```

Comment:
While I'm not opposed to improving code style (I assume that this is automated via your editor), I'm worried that this is actually going to introduce complexity when it comes to merging upstream changes in the future.

So if possible, we should be touching as little code as possible when editing this repository.